### DISCUSSION TITLE
Add Separate Unit Property For Plural Abbreviations

### DISCUSSION THREAD

**Discussion by ILSPlatform** [angularsen/UnitsNet#1122] [Ideas]
Currently, it seems that abbreviations include both singular and plural abbreviations.  Hoping these can also be separated out into two distinct properties, unless there is already a way to designate a secondary abbreviation in addition to "Default".

Currently working on a project that has Unit name, Plural name, Short name (abbreviation), and Plural short name.  Much of the data I'm working with may only have an abbreviation.  So this library is helpful in being able to lookup by abbreviation, then get the other Unit information (Name, PluralName, etc.)  

The only thing that would make this even better would be the ability to get a corresponding Plural abbreviation.  Maybe a dictionary of some sort, so if there are multiple singular abbreviations for a Unit, they could be mapped to their corresponding plural abbreviations.   Unless there is already a way to do this?

Thanks!

**Comment by angularsen** [comment]
There is no distinction on singular/plural abbreviations, we simply add all the abbreviations that are widely used in order to support parsing units and quantity strings with these abbreviations.

In my experience, the majority of abbreviations are singular ("cm", not "cms" for centimeters).
Some have both; `h`, `hr` and `hrs` for hours.

I'm unsure about this feature.
Can you help me understand why it would be useful to distinguish singular vs plural abbreviations?
Also, won't there be many units with no plural abbreviations?
